sky/jobs/state.py

```diff
@@ -2331,3 +2337,109 @@ def get_all_job_ids_by_name(name: Optional[str]) -> List[int]:
         rows = session.execute(query).fetchall()
         job_ids = [row[0] for row in rows if row[0] is not None]
         return job_ids
+
+
+@_init_db_async
+async def get_task_logs_to_clean_async(retention_seconds: int,
+                                       batch_size) -> List[Dict[str, Any]]:
+    """Get the logs of job tasks to clean.
+
+    The logs of a task will only cleaned when:
+    - the task is in terminal state (SUCCEEDED, FAILED, CANCELLED)
```

Comment:
Can we remove the example list here, since it's not exhaustive?
```suggestion
    - the task is in terminal state
```